src/main/java/com/airbnb/billow/ARNGenerator.java

```diff
@@ -33,7 +34,10 @@ private static String loadAccountNumber(AmazonIdentityManagementClient iamClient
         String userARN = null;
 
         try {
-            GetUserResult getUserResult = iamClient.getUser();
+            final String USER_NAME = "MikeCurtis";
```

Comment:
Since this account number rarely changes, how about putting it in data bag? Doing so can avoid this lame hacky way of retrieval.